From eeefce16fc1fd528e6f9f3fa736ff24a8f78bd3d Mon Sep 17 00:00:00 2001
From: Robert Chiras <robert.chiras@nxp.com>
Date: Fri, 5 Jan 2018 11:59:01 +0200
Subject: [PATCH 3179/5242] MLK-17280-2: arm64: dts: fsl-imx8qxp-mek: Add DSI
 panel reset-gpio

commit  e922c0fb6a003f9353cc764c6f99f00f551eabd3 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add the reset-gpio property for the DSI panels so that power ON/OFF can
work properly.

Signed-off-by: Robert Chiras <robert.chiras@nxp.com>
Reviewed-by: Laurentiu Palcu <laurentiu.palcu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../dts/freescale/fsl-imx8qxp-mek-dsi-rm67191.dts  |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek-dsi-rm67191.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek-dsi-rm67191.dts
index 4e0c3c0..81600f7 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek-dsi-rm67191.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek-dsi-rm67191.dts
@@ -28,6 +28,7 @@
 	panel@0 {
 		compatible = "raydium,rm67191";
 		reg = <0>;
+		reset-gpio = <&pca9557_a 6 GPIO_ACTIVE_HIGH>;
 		dsi-lanes = <4>;
 		panel-width-mm = <68>;
 		panel-height-mm = <121>;
@@ -59,6 +60,7 @@
 	panel@0 {
 		compatible = "raydium,rm67191";
 		reg = <0>;
+		reset-gpio = <&pca9557_b 7 GPIO_ACTIVE_HIGH>;
 		dsi-lanes = <4>;
 		panel-width-mm = <68>;
 		panel-height-mm = <121>;
-- 
1.7.9.5

